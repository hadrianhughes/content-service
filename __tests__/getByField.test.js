const getByField = require('../lib/getByField');

jest.mock('../__mocks__/prismic-javascript.js');

const mockData = require('../__mock-data__/doc.json');
const mockApi = {
  query: q => new Promise ((resolve, reject) => {
    resolve(q);
  })
};

describe('getByField function', () => {
  it('Should return a promise which resolves with an object', () => {
    expect(getByField(mockApi, 'type', 'field', 'value').resolves).toMatchSnapshot();
  });
});

