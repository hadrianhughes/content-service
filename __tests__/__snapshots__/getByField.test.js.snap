// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getByField function Should return a promise which resolves with an object 1`] = `undefined`;
